### DISCUSSION TITLE
Verifcation link returns 400

### DISCUSSION THREAD

**Discussion by alfredjophy** [pocketbase/pocketbase#3927] [Q&A] [answered]
When opening the verification link, I am getting a 400. 
On checking the logs

```json
{
  "errorMessage": "SyntaxError: invalid character 'u' looking for beginning of value at parse (native)"
}
```

**Comment by ganigeorgiev** [comment]
I suspect that the submitted request body is not a properly formatted json (or it is missing `content-type` header).

Could you provide an example how you are sending the request?

**Reply by ganigeorgiev** [reply, marked as answer]
On second read, it could be also related to an incorrect operation if you use the JS hooks. If that's the case, double check your hooks code or share a code sample to try to reproduce it.

**Reply by alfredjophy** [reply]
The request is sent by the built-in page. 

I have quite a few custom hooks, I will  look into it and let you know.

**Comment by alfredjophy** [comment]
Closing. It was an issue with a hook. Thanks
